testsuite/python310.py

```diff
@@ -0,0 +1,12 @@
+var, var2 = 1, 2
+#: Okay
+match (var, var2):
+    #: Okay
+    case [2, 3]:
+        pass
+    #: Okay
+    case (1, 2):
+        pass
+    #: Okay
```

Comment:
the `#:` is supposed to split separate cases -- you shouldn't need anything except the first comment (also please include the `var1, var2` in the example)

Reply:
👌🏻 Didn't know that. Is there a way to run the test case only for Python 3.10+? Especially with the first change you suggested, the tests will fail without it.

Reply:
the file naming is supposed to handle that, but it looks like this needs to have a `+` added to the second digit group here: https://github.com/PyCQA/pycodestyle/blob/c4460cb9dffbd9b7ee000dc31388d5f45fe484a0/testsuite/support.py#L172